docs/csharp/language-reference/compiler-messages/cs1997.md

```diff
@@ -9,7 +9,7 @@ helpviewer_keywords:
 ---
 # Compiler Error CS1997
 
-Since is an async method that returns `Task`, a return keyword must not be followed by an object expression. Did you intend to return `Task<T>`?
+Since `method` an async method that returns `Task`, a return keyword must not be followed by an object expression. Did you intend to return `Task<T>`?
```

Comment:
```suggestion
Since `method` is an async method that returns `Task`, a return keyword must not be followed by an object expression. Did you intend to return `Task<T>`?
```